students/17th/team1/gyuboshim/westagram/user/views.py

```diff
@@ -0,0 +1,68 @@
+from django.shortcuts import render
+
+import json
+from django.views   import View
+from django.http    import HttpResponse, JsonResponse
+
+from .models        import User
+
+PASSWORD_LEN_LIMIT = 8
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data            =   json.loads(request.body)
+            user_data       =   data['user']
+            phone_number    =   user_data['phone_number']
+            email_adress    =   user_data['email_adress']
+            name            =   user_data['name']
+            nickname        =   user_data['nickname']
+            password        =   user_data['password']
+
+
+            if user_data['email_adress'] and user_data['password']:
+                
+                if user_data['email_adress'] == User.objects.filter(email_adress = email_adress):
+                    return JsonResponse({'message':'CHANGE_E-MAIL_ADRESS'}, status = 409)
```

Comment:
응답이 너무 친절하네요 규보님. 
```suggestion
                    return JsonResponse({'message':'EMAIL_ALREADY_EXISTS'}, status = 409)
```
근데 바꾸라고 하는것 보다는 이미 존재하니까 존재한다고 말해주는게 맞습니다.

Reply:
위의 수정사항을 기준으로 아래의 코드들도 수정 부탁드립니다.

Reply:
네 확인했습니다